test/test_splash.py

```diff
@@ -0,0 +1,94 @@
+import splash
```

Comment:
These aren't really tests, but rather evaluations, might make more sense to move to a folder called evaluation or temporary?

Reply:
I added a few more SPLASH verifications in the unit tests, but they compute the SPLASH from the `MsmsSpectrum` object rather than the SPLASH that's displayed, because otherwise it's impossible to reconcile float32/float64 mismatches. Quite a bad implementation from the SPLASH devs imo.